### ISSUE TITLE
Bedrock deletion with EMC-Powered Build Wand

### ISSUE THREAD

**Issue by Hadies-Alchemist** [JPANv2/EMCBaubles#10]
Hi, i recently took to using the above mentioned wand on the erase setting to delete dungeons that i either concurred or were destroyed by thing like cave/world generation. I have a flat bedrock mod, thats kind of important. I went down to bedrock and set that block as the start position, moved and hit the second position to erase the dungeon and several dozen chunks. this took a while. 
When I could finally do things in the world again, i saved and quit and tried loading twice before it worked. When I finally loaded into the world I hopped into the resulting hole and no bedrock. I had to use my waypoint mod to escape certain death as the lag was too bad to fly. If you need any information comment and I'll get back to you.

**Comment by JPANv2** [comment]
Yeah, on all the years of using this mod in several versions of minecraft, It never occured to me to test it on bedrock. So, in the latest version for 1.16 I fixed that by disabling bedrock replacing and erasing. In the preliminary tests it works out, but I'll probably test it some more before closing this issue.

**Comment by Hadies-Alchemist** [comment]
I know the feeling of overlooking something like this, I'm a writer and I overlooked a detail that made me feel dumb when it was pointed out.
